src/Makefile.am

```diff
@@ -224,7 +224,7 @@ obj/build.h: FORCE
 	  $(abs_top_srcdir)
 libbitcoin_util_a-clientversion.$(OBJEXT): obj/build.h
 
-xversionkeys.h: FORCE
+xversionkeys.h: $(top_srcdir)/contrib/devtools/xversionkeys.py $(top_srcdir)/src/xversionkeys.dat
```

Comment:
I tried this again on native win, unfortunately it doesn't work...

Reply:
@Greg-Griffith : Does the patch work for you?

Reply:
No